### PR TITLE
DO NOT MERGE: Upgrade webpack

### PR DESCRIPTION
**Pull request by alexef**
- install in makefile
- bump webpack, webpack-cli, webpack-dev-server and html-webpack-plugin to latest
- fix webpack config

for some reason the build list css is now broken for me. I can't seem to figure out how to fix it

### CODE REVIEW

**Comment by alexef** [comment]
fyi: @ksvital

File: web/package.json [hunk 1]
```diff
@@ -98,7 +98,9 @@
     "sinon-chai": "^2.13.0",
     "style-loader": "^0.18.2",
     "url-loader": "^0.5.9",
-    "webpack": "^3.4.1",
-    "webpack-dev-server": "^3.1.11"
+    "webpack": "5.24.1",
+    "webpack-cli": "4.5.0",
+    "webpack-dev-server": "3.11.2"
+
```
Comment (alexef) [inline web/package.json:104]:
```suggestion
```

**Comment by alexef** [comment]
Potentially related, I have this in the console: ```Uncaught Error: only one instance of babel-polyfill is allowed
    at eval (VM953 index.js:10)
    at Object../node_modules/babel-polyfill/lib/index.js (bundle.8b9958ef51231b0c57a8.js:51)
    at __webpack_require__ (bundle.8b9958ef51231b0c57a8.js:4750)
    at bundle.8b9958ef51231b0c57a8.js:4827
    at bundle.8b9958ef51231b0c57a8.js:4844```

**Comment by laszlocph** [comment]
As of now master has a working local dev env, as per: https://github.com/laszlocph/woodpecker/pull/146#issuecomment-786476059

But the toolchain must be brought to 2021 soon.

I checked this PR and it seems that while it compiles and the dev server comes up the CSS is messed up. I also see the `babel-polyfill` error.

image

I'm not a webpack guru, but seems there is quite a few things configured there and probably things are done differently with the latest webpack.

Do you know how to fix the CSS issue?

Meanwhile I experiment a bit with the latest webpack toolchain too.

**Comment by alexef** [comment]
> Do you know how to fix the CSS issue?

I don't. Not sure if it's caused by missing CSS or missing JS, but will try to get help from the team. I can see the same broken layout.

**Comment by alexef** [comment]
closing this for now, as resolving conflicts would be more troublesome than starting over. 

will try to get help and re-open the PR with a fully working solution

**Pull request closed without merging**